src/components/CioQuiz/quizApiReducer.ts

```diff
@@ -91,9 +93,15 @@ export default function apiReducer(
         selectedOptionsWithAttributes: action.payload?.quizResults.attributes,
       };
     }
+    case QuizAPIActionTypes.SET_QUIZ_RESULTS_CONFIG: {
+      return {
+        ...state,
+        resultsConfig: action.payload?.quizResultsConfig.results_config,
+      };
+    }
 
     case QuizAPIActionTypes.RESET_QUIZ:
-      return initialState;
+      return { ...initialState, resultsConfig: state.resultsConfig };
```

Comment:
Why are we not resetting `resultsConfig` isn't it possible it can change?

Reply:
[7a5dd35](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/155/commits/7a5dd35b63e66ffec87f0e794ff0d8b6f695793a)